README.md

```diff
@@ -90,7 +153,7 @@ You may also wish to glance at the (much shorter) [Replication README](app/jobs/
 RAILS_ENV=production bundle exec rake ...
 ```
 
-### Rails Console
+## Rails Console
```

Comment:
I don't think we need to tell people how to use Rails Console.

Reply:
Ok I've struck that section and also the general description of Rake tasks.